src/network/networkd-network.c

```diff
@@ -131,6 +132,10 @@ static int network_load_one(Manager *manager, const char *filename) {
         if (!network->fdb_entries_by_section)
                 return log_oom();
 
+        network->prefixes_by_section = hashmap_new(NULL);
```

Comment:
Replying to self: the hashmap should be initialized with &network_config_hash_ops instead of NULL so that the sections are properly found.